src/Routes.tsx

```diff
@@ -0,0 +1,28 @@
+import { Redirect, Route, Switch } from "react-router-dom";
+import Recipes from "./pages/Recipes/Recipes";
+import RecipeDetail from "./pages/RecipeDetail/RecipeDetail";
+import RecipeForm from "./pages/RecipeForm/RecipeForm";
+
+const Routes = () => {
+  return (
+    <Switch>
+      <Route path="/recipes/create">
+        <RecipeForm mode="CREATE" />
+      </Route>
+      <Route path="/recipes/:recipeId/edit">
+        <RecipeForm mode="UPDATE" />
```

Comment:
This is a paradigm I've used previously where a form has a "CREATE" mode and an "UPDATE" mode. There are benefits and drawbacks to this approach, so I'm not sure if we tend to treat create and update forms separately at TK?